src/Samples/UiBoundCmdParam/UiBoundCmdParam.fsproj

```diff
@@ -4,6 +4,8 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <UseWpf>true</UseWpf>
+    <!--Turn on warnings for unused values (arguments and let bindings) -->
+    <OtherFlags>$(OtherFlags) --warnon:1182</OtherFlags>
```

Comment:
I'd like all of these reverted in samples

Reply:
Done